onnx2torch/node_converters/reduce.py

```diff
@@ -0,0 +1,102 @@
+__all__ = ['OnnxReduce']
+
+from typing import List
+from typing import Optional
+
+import torch
+from torch import nn
+
+from onnx2torch.common import OperationConverterResult
+from onnx2torch.common import onnx_mapping_from_node
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+
+_TORCH_FUNCTION_FROM_ONNX_TYPE = {
+    'ReduceMax': torch.max,
+    'ReduceMean': torch.mean,
+    'ReduceSum': torch.sum,
+    'ReduceMin': torch.min,
+    'ReduceProd': torch.prod,
+}
+
+
+class OnnxReduce(nn.Module):
```

Comment:
ReduceSum must be realized in separate classes (Dynamic/Static version like squeeze)

Reply:
Done.